hasher-matcher-actioner/hmalib/common/models/tests/test_bank_member_with_key.py

```diff
@@ -14,41 +17,100 @@
 
 class BankMemberWithKeyTestCase(BanksTableTestBase, unittest.TestCase):
```

Comment:
(ignorable): I've said this in other PRs, but py.test is strictly better than unittest.TestCase, and there exists a conversion to an equivalent py.test with better error messages from any existing unittest. I think I'd hard block you for py-tx (which I had gone through a conversion at some point earlier this year), but will leave as an ignore here

Reply:
Thanks! I'm not fully onboard with the functions are better than unittest.TestCase paradigm. Probably because I've not done enough work to form an opinion.

However, in this case, we have done a ton of work into designing a test classes hierarchy that makes testing dynamodb models really easy. Unless we go through a rigorous conversion in hmalib too, test classes are here to stay.

Reply:
If you weren't using that hierarchy, my opinion is it's worth it to convert having gone through the exercise once.